.github/CONTRIBUTING.md

```diff
@@ -4,16 +4,6 @@
 
 Cockatrice is written in C++-03, so do not use C++11 constructs such as `auto`.
```

Comment:
By now we enabled using a c++11 compiler, but there is no code using c++11 specific syntax yet.
The first thing i'd do after switching to Qt5 would be to refactor QObject::connect() calls in order to use use compile-time checking (instead of runtime checking) and lambdas where the code would benefit (https://wiki.qt.io/New_Signal_Slot_Syntax#Connecting_in_Qt5)